src/api/utils.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os.path
+
+
+def load_json(file_path):
+    """
+    Reads from JSON file and returns data.
+    If file not found, creates a new one.
+    """
+    if not os.path.exists(file_path):
```

Comment:
This is an error. If a helper is asked to read data from a file, the file should exist. The utility should not silently return empty data if the file itself doesn't exist.

Reply:
It was done to make the `db.json` available in the first run, but maybe a setup instruction can be done to make it available. Then another case of `FileNotFoundError` can be thrown.

Reply:
I think when you try to open a path "FileNotFoundError" gets thrown automatically, you might not need to throw it explicity.

Reply:
Sorry, I mean the calling function of the util function, should it return a `404: No professionals found` for empty database or just return a empty json?

Reply:
> This is an error. If a helper is asked to read data from a file, the file should exist. The utility should not silently return empty data if the file itself doesn't exist.

I have removed the part where it was creating a new `db.json` if the file was not found, that can be taken care of while setup, and the error results in a `500` error.